.github/workflows/pytest.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     strategy:
       matrix:
         os: [ "windows-latest", "ubuntu-latest" ] #, "macos-latest"]
-        python-version: [ "3.8", "3.9", "3.10" ]
+        python-version: [ "3.9", "3.10" ]
```

Comment:
What about extending the scope of the PR and adding 3.11 support here?

Reply:
We have [this PR](https://github.com/waltsims/k-wave-python/pull/206) for that. It is currently failing due to the uff dependency. What will be alleviated with https://github.com/waltsims/k-wave-python/pull/229

Reply:
Both PRs are merged now. I updated the branch with `master` and resolved the conflicts. Maybe it is time to merge? :)

Reply:
And tested passed

Reply:
Pre or post release? We won't make any python 3.8 braking changes yet will we?

Reply:
Good point, no breaking changes on my side. We can do it post release also.